scripts/plot_network.py

```diff
@@ -236,30 +240,21 @@ def split_costs(n):
         texts.append(text)
 
     ax.set_ylabel("Average system cost [Eur/MWh]")
-    ax.set_ylim([0,80]) # opts['costs_max']])
-    ax.set_xlim([0,1])
+    ax.set_ylim([0, 80]) # opts['costs_max']])
+    ax.set_xlim([0, 1])
     #ax.set_xticks([0.5])
     ax.set_xticklabels([]) #["w/o\nEp", "w/\nEp"])
     ax.grid(True, axis="y", color='k', linestyle='dotted')
 
 
 if __name__ == "__main__":
     if 'snakemake' not in globals():
-        from vresutils.snakemake import MockSnakemake, Dict
-        from snakemake.rules import expand
-
-        snakemake = Dict()
-        snakemake = MockSnakemake(
-            path='..',
-            wildcards=dict(network='elec', simpl='', clusters='90', lv='1.25', opts='Co2L-3H', attr='p_nom', ext="pdf"),
-            input=dict(network="results/networks/{network}_s{simpl}_{clusters}_lv{lv}_{opts}.nc",
-                    tech_costs="data/costs.csv"),
-            output=dict(only_map="results/plots/{network}_s{simpl}_{clusters}_lv{lv}_{opts}_{attr}.{ext}",
-                        ext="results/plots/{network}_s{simpl}_{clusters}_lv{lv}_{opts}_{attr}_ext.{ext}")
-        )
+        snakemake = mocksnakemake('plot_network', network='elec', simpl='',
```

Comment:
```suggestion
        from _helpers import mocksnakemake
        snakemake = mocksnakemake('plot_network', network='elec', simpl='',
```

to stay consistent with the other files